include/openPMD/benchmark/mpi/MPIBenchmark.hpp

```diff
@@ -180,9 +182,13 @@ namespace openPMD
 
             /**
          * Execute a single read benchmark.
-         * @tparam Clock Clock type to use.
+         * @tparam T Clock type to use.
```

Comment:
that's wrong... it's the data type? The `Clock` `tparam` must move up as part of the `BenchmarkExecution` doxygen block.